core/trino-main/src/main/java/io/trino/execution/SqlTaskExecution.java

```diff
@@ -321,6 +323,11 @@ public void addSources(List<TaskSource> sources)
         }
     }
 
+    public void addDynamicFilter(Map<DynamicFilterId, Domain> dynamicFilterDomains)
```

Comment:
remove this method, see https://github.com/trinodb/trino/pull/5183/commits/d2f820cfabf407a117ddd90110e1f091ce583a90#r660572226